TextAnalyzer/Commands/DownloadFileCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TextAnalyzer
+{
+    class DownloadFileCommand : ICommand
+    {
+		public bool Reportable => false;
+
+		public string Description => "Download file from internet";
```

Comment:
Wcięcia.

Reply:
W moim IDE jedyny problem jaki w tym miejscu zauważyłem to brak przejścia do nowej linii przy wprowadzonej klamerce. Naprawiłem to, ale z samą indentacją nie widziałem problemów.